_posts/en/newsletters/2022-04-27-newsletter.md

```diff
@@ -0,0 +1,240 @@
+  feelings about the subject---whether in favor or opposed to activating
+  CTV in the next few months---read through the discussions and consider
+  your next actions.  Any person or service provider who doesn't want to
+  become involved at this time can continue running their normal full
+  node software for at least the next two weeks.  We will provide
+  updates on the subject in future newsletters.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How was the generator point G chosen in the secp256k1 curve?]({{bse}}113116)
+  Pieter Wuille notes that while the exact rationale behind choosing constant [generator
+  point G][se 29904] is not publicly known, an unusual property may hint of its construction.
+
+- [What is the maximum, realistic P2P message payload size?]({{bse}}113059)
+  0xb10c asks if there is a valid [P2P message][p2p messages] of `MAX_SIZE` (32MB). Pieter
+  Wuille explains that `MAX_PROTOCOL_MESSAGE_LENGTH` ([4MB][bitcoin protocol
+  4mb], increased from [2MB][Bitcoin Core #5843] as
+  [part of segwit][Bitcoin Core #8149]) is what actually restricts the size of incoming messages to
+  prevent denial-of-service attacks.
+
+- [Is there evidence for lack of stale blocks?]({{bse}}113413)
+  Lightlike references a historical chart of block propagation times from the
+  [KIT statistics][] website and points out [compact block relay][topic compact
+  block relay] ([BIP152][]), implemented initially in [#8068][Bitcoin Core
+  #8068], as a contributor to decreased [stale block][se 5866] frequency over time.
+
+  {:.center}
+  ![Block Propagation Delay History chart](/img/posts/2022-04-block-propagation-delay.png)
+
+- [Does a coinbase transaction's input field have a VOUT field?]({{bse}}113392)
+  Pieter Wuille outlines the requirements for a coinbase transaction's input: the
+  prevout hash must be
+  `0000000000000000000000000000000000000000000000000000000000000000`, the prevout
+  index must be `ffffffff`, it must have a 2-100 byte `scriptSig` length,
+  and since [BIP34][] the `scriptSig` must also start with the block height.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 23.0][] is a candidate for the next major
+  version of this predominant full node software.  The [release
+  notes][bcc23 rn] list multiple improvements.  FIXME:harding:improve
```

Comment:
We should definitely mention that 23.0 creates Taproot descriptors for new wallets. :monocle_face: